src/Views/Components/AdminUI/api.ts

```diff
@@ -0,0 +1,35 @@
+import {useState} from 'react';
+
+const PostRequest = (endpoint: string, apiNonce: string, successMessage: string, errorMessage: string) => {
+    const [result, setResult] = useState({type: null, message: ''});
```

Comment:
Is `PostRequest` used as a component, a hook, or a function? We're using a hook here, so it must be one of the first two. I would think a hook, in which case the name should be `usePostRequest`.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6752/commits/9b6b50ee131b40b30fd730d50c2f5debff76df09